### PR TITLE
Avoid unnecessary `require`s

### PR DESCRIPTION
**Pull request by taylor1791**
We encountered a problem with `webworkify-webpack`. By `require`ing every module, it is forcing all modules to be initialized. Some of our modules have assumptions about when they are initialized. In particular, they have to be initialized after we use something that uses `webworkify-webpack` as a dependency.

To fix this, I purpose that we avoid `require`ing the modules by doing a simple check to see if the module "looks like" the right one. Making `wrapperFuncString.indexOf(fnString) > -1` a necessary condition for the require address this issue. The subsequent logic is updated to behave appropriately since the conditions are now reversed.

This fixes the problem and does not change existing behavior as far as I know. This can be considered a breaking change. However, it is one that reverts the behavior to what I and most users expect.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,24 +24,29 @@ var webpackBootstrapFunc = function(modules) {
 }
 
 module.exports = function (fn) {
+    var fnString = fn.toString();
```
Comment (taylor1791) [inline index.js:27]: Moving this line because it avoids stringifying the function numerous times.

**Comment by borisirota** [comment]
Thanks ! available in version 1.1.0.

Please take a look at the [caveat](https://github.com/borisirota/webworkify-webpack#caveat) I added to the README.

**Comment by taylor1791** [comment]
Ah! Good thinking about the caveat. I forgot about that option in webpack.

**Comment by yocontra** [comment]
Is there any way to get around this? Builds are now 60+ seconds when they used to be 400ms because of this :(

**Comment by borisirota** [comment]
@contra please give more info. `webworkify-webpack` is a runtime worker builder. It shouldn't affect the webpack build.

**Comment by yocontra** [comment]
@borisirota You can't use efficient sourcemaps after this was merged, there's a note in the README about it. It does affect webpack build perf by placing that limitation on anyone who uses it.

**Comment by borisirota** [comment]
@contra you are right and you can get around this by using v1.0.6.

**Comment by borisirota** [comment]
@contra in the new v2 there is no limitation on the `devtool` options (eval can be used as well)

**Pull request merged**